### PR TITLE
Updated C28750

### PR DESCRIPTION
**Pull request by MugBergerFries**
Added more context, an example, and modified the wording and table to match the formatting of my other PRs

### CODE REVIEW

File: windows-driver-docs-pr/devtest/28750-banned-istrlen-usage.md [hunk 1]
```diff
@@ -9,49 +9,47 @@ f1_keywords:
...
+The correct mitigation is to use a safer string length function (usually strlen, wcslen, _tcslen). However, while you review the lstrlen changes, you should confirm that the string buffer is coming from trusted code. If you are dealing with untrusted data, you should instead switch from the strlen family of functions to the strnlen family (or StringCchLength family), which will ensure they don't go past the bounds of the untrusted data block. 
+
+Note that unlike lstrlen, none of the replacements catch exceptions. Also, lstrlen handles NULL pointers while the replacements do not, so an explicit NULL check is required when replacing lstrlen with strlen or strnlen (if NULL pointers are possible at that point in the code).  
+
+A list of all banned functions covered by this error and recommended replacements (for both trusted and untrusted data) can be found after the following example: 
+
+## Example
+
+The following code generates this warning: 
+```
```
Comment (MichaelSquires) [inline windows-driver-docs-pr/devtest/28750-banned-istrlen-usage.md:26]:
```suggestion
```c
```

File: windows-driver-docs-pr/devtest/28750-banned-istrlen-usage.md [hunk 2]
```diff
@@ -9,49 +9,47 @@ f1_keywords:
...
+int example_func(char* in)
+{ 
+    int size = lstrlen(in);
+    return size; 
+} 
+```
+This is due to the use of the unsafe function lstrlen. To fix this issue, we can use the strlen as the replacement, making sure to check if the pointer is NULL: 
+```
```
Comment (MichaelSquires) [inline windows-driver-docs-pr/devtest/28750-banned-istrlen-usage.md:34]:
```suggestion
```c
```

File: windows-driver-docs-pr/devtest/28750-banned-istrlen-usage.md [hunk 3]
```diff
@@ -9,49 +9,47 @@ f1_keywords:
...
+        // handle error.
+    }
+} 
+```
+
+## Banned Functions
+
+| Banned API | Trusted data replacement(s) | Untrusted data replacement(s) |
+| -----------|----------------|--------------|
+|```lstrlen```| ```_tcslen``` | ```_tcsnlen```, ```StringCchLength``` |
```
Comment (MichaelSquires) [inline windows-driver-docs-pr/devtest/28750-banned-istrlen-usage.md:50]: For inline code like these, you can use single backticks around the words. Triple backticks are only required for multi-line code.

(I'm not suggesting you change these, just for future reference...)

**Review by MichaelSquires — CHANGES_REQUESTED** [review]

File: windows-driver-docs-pr/devtest/28750-banned-istrlen-usage.md (hunk 1 above)
Comment (MugBergerFries) [reply windows-driver-docs-pr/devtest/28750-banned-istrlen-usage.md]: Added cpp context!

File: windows-driver-docs-pr/devtest/28750-banned-istrlen-usage.md (hunk 3 above)
Comment (MugBergerFries) [reply windows-driver-docs-pr/devtest/28750-banned-istrlen-usage.md:50]: Roger that, will do!

**Review by Rastaban — APPROVED** [review]

**Review by MichaelSquires — APPROVED** [review]

**Comment by MugBergerFries** [comment]
@DOMARS This is approved and ready to be merged

**Pull request merged**